### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by norbiboy1**
- Align background professionally;
- Responsiveness and style for the two devices - mobile and desktop;
- Validation of contact form.

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
# *Status: Required Changes* :recycle: 

Hi @norbiboy1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-  Good use of Github Flow. :heavy_check_mark: 
-  Good commit messages. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Good job on implement some features to make your desktop version. You've really done great job. However, there are some issues you need to attend to before moving on. Your desktop implementation broke some of your mobile version. Kindly take a look at the below snapshots and fix all the areas where needed.

### Your Design hamburger for mobile

your-hamburger-icon

#
### Figma Design hamburger

figma-hamburger-icon
 
#

### Your form design on mobile

- [ ] Kindly consider giving your form section some padding left and right so as to make it look great and inline with your chosen Figma template.

form-design-yours


#

### Figma form design on mobile

form-design-figma

#

### Your footer Icons position on desktop

- [ ] Kindly consider styling your footer icons to be at the center of the page in order to be inline with the figma template you chose.

your-footer-icons

#

### Figma Design Icon position on desktop

figma-footer-icons

#

### Your design icons position on mobile

- [ ] Kindly consider reducing the space between your footer icons and your email address and leave some margin bottom to the section so as to be inline with the figma template you chose.

your-footer-space

#

### Figma Design icons on mobile

figma-footer-space

File: portfolio.css [hunk 1]
```diff
@@ -1,1025 +1,1825 @@
...
+  }
+
+  footer {
+    display: none;
+  }
+}
+
+/* Media queries for the desktop version breakthrough */
+
+@media only screen and (min-width: 768px) and (max-height: 3846px) {
```
Comment (CollinsTatang) [inline portfolio.css:1086]: - [ ] Please kindly fix the mobile responsiveness as mentioned by the previous reviewer [Check Figmal Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
**Your Design**
checkReview

**Figmal Design**
image

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @norbiboy1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] Please kindly fix the mobile responsiveness as mentioned by the previous reviewer

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**